ui/job-view/PushList.jsx

```diff
@@ -82,6 +87,15 @@ export default class PushList extends React.Component {
         }
       }
     );
+
```

Comment:
this is here, rather than ``PushJobs`` because in ``PushJobs`` it will happen for each push in our list, whereas here, it happens only once for all pushes.